stubparser/src/org/checkerframework/stubparser/ast/visitor/DumpVisitor.java

```diff
@@ -221,9 +221,22 @@ private void printMembers(List<BodyDeclaration> members, Object arg) {
 
     private void printMemberAnnotations(List<AnnotationExpr> annotations, Object arg) {
         if (annotations != null) {
+            for (Iterator<AnnotationExpr> i = annotations.iterator(); i.hasNext();) {
+                AnnotationExpr a = i.next();
+                a.accept(this, arg);
+                if (i.hasNext()) {
+                    printer.print(" ");
+                }
+            }
+            printer.printLn();
+        }
+    }
+
+    private void printReceiverAnnotations(List<AnnotationExpr> annotations, Object arg) {

```

Comment:
I needed to eliminate the trailing space.  The treatment of whitespace is the main difference among the various print*Annotations methods.